app/jobs/send_invitation_email_job.rb

```diff
@@ -0,0 +1,8 @@
+class SendInvitationEmailJob < ApplicationJob
+  queue_as :default
+
+  def perform(user, password)
```

Comment:
У воркери треба передавати якнайпростіші об"єкти: числа, стрінги, масиви, хеші з простими об"єктами.
```suggestion
  def perform(user_id, password)
```